modules/messages/domain/tool-result-output.ts

```
@@ -0,0 +1,18 @@
+		return output;
+	}
+
+	const outputRecord = output as Record<string, unknown>;
+	switch (outputRecord["type"]) {
+		case "json":
+		case "text":
+		case "error-json":
+		case "error-text":
+		case "content":
+			return outputRecord["value"];
+		case "execution-denied":
```

Biome (2.4.10)

[error] Useless case clause.

(lint/complexity/noUselessSwitchCase)